app/src/main/java/mozilla/lockbox/presenter/ItemMutationPresenter.kt

```diff
@@ -116,23 +146,45 @@ abstract class ItemMutationPresenter(
             .addTo(compositeDisposable)
     }
 
+    fun blurredOnce(focusChange: Observable<Boolean>): Observable<Boolean> {
+        var focusOnce = false
+        var blurredOnce = false
```

Comment:
Too many uses of "blurredOnce". Can improve the naming here

Reply:
Renamed to `focusLostOnce`, and commented the method.